src/lamplib/tasks/run_tests.py

```diff
@@ -110,7 +110,11 @@ def resmoke_test(env, suites, mongo_dir, is_cnats):
         # Default download location for MongoDB binaries.
         env["PATH"] += ":" + os.path.join(mongo_dir, "bin") + ":" + mongo_dir
 
-    evg_venv_dir = os.path.join(workdir, "build", "venv")
+    if 'LAMP_VENV_DIR' not in os.environ:
+        raise ValueError("The venv directory is required for resmoke and does not exist, "
+                         "please ensure you're not running Genny with the --run-global flag")
+
+    evg_venv_dir = os.path.join(os.environ['LAMP_VENV_DIR'], "venv")
```

Comment:
Right after this, an `os.path.exists(evg_venv_dir)` check is probably a good call.

Reply:
Similar to above, the lack of a valid virtualenv directory is an internal genny error and should be something we're able to catch in our tests, not something the user has to see. The purpose of the error handling here is to capture user mistakes.